leobot_web_server/scripts/web_server.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env nodejs
```

Comment:
please use this approach https://github.com/RethinkRobotics-opensource/rosnodejs_examples/blob/kinetic-devel/scripts/listener.js#L43